sdk/servicebus/service-bus/samples/javascript/README.md

```diff
@@ -12,7 +20,8 @@ These sample programs show how to use the JavaScript client libraries for Azure
 | [scheduledMessages.js][scheduledmessages]                           | uses the scheduleMessage() function to schedule messages to appear on a Service Bus Queue/Subscription at a later time  |
 | [session.js][session]                                               | sends/receives messages to/from session enabled queues/subscriptions in Service Bus                                     |
 | [browseMessages.js][browsemessages]                                 | uses the peek() function to browse a Service Bus                                                                        |
-| [usingAadAuth.js][usingaadauth]                                     | creates a ServiceBusClient that authenticates using AAD credentials                                                     |
+| [serviceprincipallogin.js][serviceprincipallogin]                   | creates a namespace using aad token credentials obtained from using service principal secrets                           |
```

Comment:
What about the fact that all these links will land to the master branch anyway?

Reply:
No, I have updated them below
![image](https://user-images.githubusercontent.com/10452642/77691136-a17c2500-6f61-11ea-8f0f-28fc5829eec1.png)